src/page-content/navigation/navigation-header.tsx

```diff
@@ -135,7 +135,7 @@ export const mobileMenu: Pick<NavProps, 'mobileMenu'> = {
           },
           {
             title: 'Pricing',
-            to: '/pricing'
+            to: '/pricing/'
```

Comment:
why this change?

Reply:
I was not sure if we should have trailing slashes or not, it seemed more had it than not, so I added it to be consistent. 
But they should be without right?
```suggestion
            to: '/pricing'
```

Reply:
Don't see any with, probably both are handled but we should be consistent by not having them IMO.